synapse/handlers/sync.py

```diff
@@ -1010,6 +1039,13 @@ async def compute_state_delta(
                             ),
                         )
 
+            # At this point, `state_ids` includes the memberships of all event senders
+            # in the timeline when `lazy_load_members` is enabled. This is because we
+            # may not have sent the memberships in a previous sync.
+
+            # When `include_redundant_members` is on, we send all the lazy-loaded
+            # memberships of event senders. Otherwise we make an effort to prune the set
+            # of memberships we send:
```

Comment:
```suggestion
            # memberships of event senders. Otherwise we make an effort to limit the set
            # of memberships we send to those that we have not already sent to this client.
```

... or something to that effect?